effekt/jvm/src/test/scala/effekt/LLVMTests.scala

```diff
@@ -29,6 +29,10 @@ class LLVMTests extends EffektTests {
     examplesDir / "llvm" / "polymorphism_map.effekt",
     examplesDir / "pos" / "type_parameters_blocks.effekt",
 
+    // Type inference during polymorphism boxing
+    examplesDir / "benchmarks" / "input_output" / "word_count_ascii.effekt",
+    examplesDir / "benchmarks" / "input_output" / "word_count_utf8.effekt",
```

Comment:
Is there an issue already? It would be horrible if this can't be run on LLVM only due to some bug.

Reply:
No, but I am too lazy to minimize these. We can always fix this later.

Reply:
Here is the function type that is failing when instantiated:

```
Function(List(),List(),List(),List(Interface(stop,List())),Data(Byte,List()))
```
which is the inferred type of

```
{ (){g7113} =>
  shift(p7114) { (){k7115} =>
    def resume3708(){h7116} = resume(k7115) { h7116(g7113) }

    resume3708({ (){stop4511} =>
      val v_r_49857117: Int398 = !i3707;
      if (infixLt175(v_r_49857117, size2454(bytes3621))) {
        val v_r_49867118: Int398 = !i3707;
        let v_r_49877119 = unsafeGet2457(bytes3621, v_r_49867118)

        v_r_49877119
      } else {
        stop4511.stop()
      }
    })
  }
}
```

Reply:
This is the function which is causing the problem:

```
def feed[T](array: Array[T]) { reader: () => Unit / read[T] } = {
  var i = 0
  try {
    reader()
  } with read[T] {
    resume {
      if (i < array.size) {
        array.unsafeGet(i)
      } else {
        do stop()
      }
    }
  }
}
```

Reply:
Fixed by f30d83a. I just forgot to bind `cps`...